### DISCUSSION TITLE
Sumeru update? (And possible 1440p support?)

### DISCUSSION THREAD

**Discussion by hitoridoga** [ZANdewanai/Genshin-Impact-Rich-Presence#4] [Ideas]
I tried it and it's nice, however it seems to be pretty broken with a 1440p monitor in my case. A sumeru update would be cool though :)

**Comment by ZANdewanai** [comment]
will work on the sumeru update now and about the 1440p monitor, i'll see what i can do, i dont have that kind of monitor myself but can just make my resolution bigger to test it

**Reply by hitoridoga** [reply]
Thank you in advance sir

**Reply by ZANdewanai** [reply]
Done, just download the v1.5 release 1440p ver
